libs/tup-components/src/layout/PageLayout/PageLayout.module.css

```diff
@@ -24,6 +24,8 @@
 .content {
   display: grid;
   grid-area: _content;
+
+  padding: var(--global-space--section);
```

Comment:
**Explanation:** Applying section space to all content, so each content need not do it itself. _This is easier to do now that Design agreed there is no need for selected nav links' purple background to bleed into the left-hand space of a section._